homeassistant/components/geniushub/__init__.py

```diff
@@ -7,15 +7,15 @@
 from homeassistant.helpers.discovery import async_load_platform
 
 REQUIREMENTS = ['geniushub==0.1']
-GENIUS_HUB = 'genius_hub'
-DOMAIN = 'geniushub'
+
+DOMAIN = 'genius_hub'
```

Comment:
The domain name should be the same as the component name.

Reply:
```suggestion
DOMAIN = 'geniushub'
```

Reply:
OK, my mistake. Resolved.